examples/boussinesq_2d_imex/playground.py

```diff
@@ -36,12 +36,12 @@
     swparams['do_LU'] = True
 
     sparams = {}
-    sparams['maxiter'] = 12
+    sparams['maxiter'] = 16
 
     # setup parameters "in time"
     t0     = 0
-    Tend   = 3000
-    Nsteps =  500
+    Tend   = 384
+    Nsteps =  128
```

Comment:
You can just increase Tend and Nsteps by constant factors, so e.g. Tend = 768 and Nsteps = 1000 or so. Only increasing Nsteps would also work, a finer resolution might help PFASST converge better.